Client/Frontend/Strings.swift

```diff
@@ -3930,10 +3930,15 @@ extension String {
             value: "Try our trusted guide to product reviews",
             comment: "Label for the header of the Shopping Experience Opt In onboarding Card (Fakespot)")
         public static let OptInCardFirstParagraph = MZLocalizedString(
-            key: "Shopping.OptInCard.FirstParagraph.Description.v120",
+            key: "Shopping.OptInCard.FirstParagraph.Description.v122",
```

Comment:
This change will result in all existing translations being deleted. Instead please keep the old string, add a new string containing only the full string for DE/FR without the link and remove the string for `OptInCardSupportedWebsites`.